packages/template.rb.template

```diff
@@ -1,73 +1,69 @@
-# It is recommended that you investigate other files in the package directory
-# to help get an idea of how existing packages are being installed through chromebrew
+#!/usr/bin/env ruby <- This line is not necessary, just for GitHub's language detection
```

Comment:
Do we need another `#` before `<-` to indicate the rest of the line is a comment?

Reply:
The shebang line won't work anyways as `env` does not support comments